src/Sql/Sql/ChangeLog.md

```diff
@@ -31,6 +31,7 @@
     - `Get-AzSqlInstanceLink`
     - `Remove-AzSqlInstanceLink`
     - `Set-AzSqlInstanceLink`
+* Added support for cross tenant and cross subscription restore operations for `Restore-AzSqlDatabase`
```

Comment:
Need to mention that it's only supported for DataWarehouse, say:

Added support for DataWarehouse cross tenant and cross subscription restore operations to `Restore-AzSqlDatabase` cmdlet.